enhancements/microshift/kubernetes-for-industrial-edge.md

```diff
@@ -0,0 +1,435 @@
+---
+title: kubernetes-for-industrial-edge
+authors:
+  - dhellmann
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@deads2k, control plane expert"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@jogeo, MicroShift contributor"
+  - "@mangelajo, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@sallyom, MicroShift contributor"
+approvers:
+  - "@derekwaynecarr"
+api-approvers:
+  - None
+creation-date: 2022-07-12
+last-updated: 2022-07-12
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9080
+  - https://issues.redhat.com/browse/USHIFT-50
+see-also:
+  - https://github.com/openshift/microshift/blob/main/docs/design.md
+---
+
+# Kubernetes for Industrial Edge
+
+## Summary
+
+This enhancement describes MicroShift, which provides an essential
+container orchestration runtime compatible with kubernetes and
+OpenShift built for Internet-of-things (IoT) and Edge computing
+scenarios that are both CPU and memory constrained.  The container
+orchestration runtime is binary compatible with OpenShift Container
+Platform, but it is not 100% API resource compatible.  It has a strict
+subset of OpenShift API resources pertinent for IoT and Edge computing
+scenarios with a strong bias to retaining traditional runtime security
+controls like SecurityContextConstraints and SELinux.
+
+Quite a lot of the text in this enhancement is taken from other
+working documents.
+
+## Motivation
+
+MicroShift addresses customer use cases with low-resource,
+field-deployed edge devices (SBCs, SoCs) requiring a minimal K8s
+container orchestration layer.
+
+MicroShift is targeting a class of devices like Xilinx ZYNQ
+UltraScale+, fitlet2, NVIDIA Jetson TX2, or Intel NUCs that are
+pervasive in edge deployments. These cost a few hundred USD, have all
+necessary accelerators and I/O on-board, but are typically not
+extensible and are highly constrained on every resource, e.g.:
+
+* CPU: ARM Cortex or Intel Atom class CPU, 2-4 cores, 1.5GHz clock
```

Comment:
"ARM Cortex" is not at all clear, there are many we can not run on and never will because they are 32 bit, such as Cortex A9 and Cortex A15, I think a better statement would be "64 Bit Arm"  also the clock speed of suitable systems will vary a lot, from 1.4Ghz to at least 2.2Ghz or more